### PR TITLE
feat: add select parent node of selected node

### PR DESCRIPTION
**Pull request by ccloli**
image

### CODE REVIEW

File: packages/core/src/models/workspace.ts [hunk 1]
```diff
@@ -865,6 +865,20 @@ export class Workspace extends EventTarget implements IWorkspace {
       });
   }
 
+  /**
+   * 选中当前选中节点的父节点
+   */
+  selectParentNode() {
```
Comment (wwsun) [inline packages/core/src/models/workspace.ts:871]: 直接放在 SelectSource ？

`selectSource.selectParent()`

File: packages/designer/src/widgets.ts [hunk 2]
```diff
@@ -48,4 +53,5 @@ registerWidget('sidebar.dataSource', DataSourcePanel);
 
 registerWidget('selectionMenu.copyNode', CopyNodeAction);
 registerWidget('selectionMenu.deleteNode', DeleteNodeAction);
+registerWidget('selectionMenu.parentNode', ParentNodeAction);
```
Comment (wwsun) [inline packages/designer/src/widgets.ts:56]: key 叫 "selectParentNode" 吧

File: packages/core/src/models/workspace.ts (hunk 1 above)
Comment (ccloli) [reply packages/core/src/models/workspace.ts]: 已在  [791e9ac](https://github.com/NetEase/tango/pull/158/commits/791e9acd1cd64f6f00e567c7a92374557e53f241) 调整

File: packages/designer/src/widgets.ts (hunk 2 above)
Comment (ccloli) [reply packages/designer/src/widgets.ts]: 已在 [791e9ac](https://github.com/NetEase/tango/pull/158/commits/791e9acd1cd64f6f00e567c7a92374557e53f241) 调整

**Pull request merged**